css-two-column-layout/styles.css

```diff
@@ -0,0 +1,73 @@
+.row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.column-full {
+  width: 100%;
+  text-align: center;
+  padding-left: 70px;
+  padding-right: 70px;
+}
+
+.column-half {
+  position: relative;
+  height: auto;
+  width: 450px;
+  padding-bottom: 50px;
+}
+
+.container {
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 1000px;
+}
+
+h1 {
+  font-size: 3rem;
+  font-family: 'Karla', sans-serif;
+  font-weight: 700;
+  color: black;
+  padding-top: 40px;
+}
+
+p {
+  font-family: 'Karla', sans-serif;
+  font-size: 1rem;
+  color: gray;
+  line-height: 1.5;
+}
+
+h3 {
+  font-family: 'Karla', sans-serif;
+  font-size: 1.2rem;
+}
+
+.image {
+  position: absolute;
+  right: auto;
+}
```

Comment:
This should be doable without `absolute` positioning 🤔 

Within each box with an image and text, I would make a new `row` with maybe a `column-third` and `column-two-thirds` for the image container and text container, respectively.